core/src/main/java/org/axonframework/eventsourcing/eventstore/RevisionBasedSnapshotJury.java

```diff
@@ -0,0 +1,42 @@
+package org.axonframework.eventsourcing.eventstore;
+
+import org.axonframework.serialization.RevisionResolver;
+import org.axonframework.serialization.SerializedType;
+
+/**
+ * A simple revision resolver based implementation of SnapshotJury. Decides whether to use a snapshot based on the current aggregate
+ * revision and the current snapshot revision. It simply does an == match and does not consider the order of revisions.
+ *
+ * @author Shyam Sankaran
+ */
+
+public class RevisionBasedSnapshotJury implements SnapshotJury {
+
+    private static final String NO_REVSION = "NO_REVSION";
```

Comment:
There is a typo in the constant name and its contents. It should be `NO_REVISION` i.o. `NO_REVSION`.

Reply:
Fixed.